specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -947,18 +1001,90 @@ model AcsChatMessageEventBaseProperties extends AcsChatEventBaseProperties {
   composeTime: utcDateTime;
 
   /** The type of the message */
-  type?: string;
+  type: string;
 
   /** The version of the message */
-  version?: int64;
+  @madeRequired(ServiceApiVersions.v2024_01_01)
+  version: int64;
+}
+
+/** Schema of common properties of all participant events */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantEventData extends AcsCallingEventProperties {
+  /** The communication identifier of the participant */
+  user?: CommunicationIdentifierModel;
+
+  /** The display name of the participant. */
+  displayName?: string;
+
+  /** The Id of the participant. */
+  participantId?: string;
+
+  /** The user agent of the participant. */
+  userAgent?: string;
+}
+
+/** Schema of common properties of all calling events */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallingEventProperties {
+  /** The communication identifier of the call startedBy */
+  startedBy: CommunicationIdentifierModel;
+
+  /** The callId of the server */
+  serverCallId?: string;
+
+  /** The group metadata */
+  group?: AcsCallingEventGroupProperties;
+
+  /** Is two party in calling event. */
+  isTwoParty?: boolean;
+
+  /** The correlationId of calling event */
+  correlationId?: string;
+
+  /** Is the calling event a room call. */
+  isRoomsCall?: boolean;
+}
+
+/** Schema of calling event group properties */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallingEventGroupProperties {
+  /** Calling Group ID. Format is Guid as string. */
+  id?: string;
+}
+
+/** Schema of calling event endedby properties */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallingEventEndedByProperties {
+  /** The communication identifier of the call ended by */
+  endedBy: CommunicationIdentifierModel;
+
+  /** The type of call ended by. */
+  type: CallEndedParticipantTypeProperty;
```

Comment:
Do we need property at the end of this type? Also, it doesn't match the model definition name. What about `CallEndedParticipantKind`?